go-controller/pkg/ovn/master.go

```diff
@@ -634,6 +643,15 @@ func (oc *Controller) ensureNodeLogicalNetwork(nodeName string, hostsubnet *net.
 		return err
 	}
 
+	// Add any service reject ACLs applicable for UDP LB
+	acls = oc.GetAllACLsForServiceLB(oc.UDPLoadBalancerUUID)
+	if len(acls) > 0 {
+		_, _, err = util.RunOVNNbctl("add", "logical_switch", nodeName, "acls", strings.Join(acls, ","))
+		if err != nil {
+			klog.Warningf("Unable to add reject ACL: %s for switch: %s", acls, nodeName)
```

Comment:
can you please capture `err` value in the Warning message so that we will know exactly as to why things failed?

Reply:
s/ACL/ACLs/ to be consistent with the error message at L633

Reply:
Can you please include TCP or UDP in the error message, so that we will know for what execution the command failed? Right now both the messages are same. one could look at the klog's filename:linenumber, but still it is better to have a different verbiage for the error message.

Reply:
yes